src/pretix/presale/templates/pretixpresale/event/waitinglist.html

```diff
@@ -3,36 +3,38 @@
 {% block title %}{% trans "Waiting list" %}{% endblock %}
 {% block content %}
     <h2>{% trans "Add me to the waiting list" %}</h2>
-    <form action="" method="post">
+    <form action="{{ request.get_full_path }}" method="post">
```

Comment:
Is there any benefit to this change? Just in case I can learn something from it :)

Reply:
AFAIK regarding browser implementation: No. But the HTML-validator complained about it. And in the A11y-audit they complained about invalid HTML here and there (that’s why I also added loads of <p>-elements).